### ISSUE TITLE
Inaccurate job duration

### ISSUE THREAD

**Issue by srid** [hercules-ci/support#31]
So my agent was down for several days, and once I fixed and restart it, the jobs that were 'waiting' ran and completed. Their duration reported _includes_ the number of days waited.

![image](https://user-images.githubusercontent.com/3998/67989295-c5347c80-fc08-11e9-97c4-8f261bd92469.png)

https://hercules-ci.com/github/srid/rib-sample/jobs/9

In practical terms that is not accurate, as the job took a matter of minutes to complete (albeit after having waited for days for the agent to come online).

**Comment by domenkozar** [comment]
Thanks for the bug report. 

It seems like the duration should really be sum of the time spent building derivations, but then we also need a timestamp when job completed. 

Does that make more sense?

**Comment by srid** [comment]
Yes. Maybe even a 'timeline' of events each having their own timestamp.
